src/LifeCycle.hs

```diff
@@ -107,6 +132,15 @@ wildIdeaCap _i = \case
     Principal        -> []
     Admin            -> []
 
+wildFrozenCap :: Idea -> Role -> [IdeaCapability]
+wildFrozenCap _i = \case
+    Student    _clss -> [CanLike, CanComment]
```

Comment:
This `CanLike` is not correct, but it didn't do any harm because the like buttons are filtered out in two places in `Frontend.Page.Idea`.  (This is not an ideal situation, but I think cleaning this up is a bigger task.  I will just remove this `CanLike` here for now.)